python/hsfs/core/statistics_engine.py

```diff
@@ -81,6 +81,22 @@ def profile_statistics(metadata_instance, feature_dataframe):
         )
         return content_str
 
+    @staticmethod
+    def profile_transformation_fn_statistics(feature_dataframe, columns):
+        if len(feature_dataframe.select(*columns).head(1)) == 0:
+            raise exceptions.FeatureStoreException(
+                "There is no data in the entity that you are trying to compute "
+                "statistics for. A possible cause might be that you inserted only data "
+                "to the online storage of a feature group."
+            )
+        content_str = engine.get_instance().profile(
+            feature_dataframe,
+            columns,
+            False,
+            True,
```

Comment:
When the `exact_uniqueness` parameter is not set, it will be on by default. I think for the transformation statistincs exact uniquness is not necessary, as it is quite costly, I would advice to turn it off:

 `content_str = engine.get_instance().profile(
            feature_dataframe,
            columns,
            False,
            True,
            False )`